src/js/dp/release-calendar.js

```diff
@@ -35,6 +35,39 @@ document.addEventListener("DOMContentLoaded", function () {
   if (findNode(".release-calendar")) {
     releaseTypeAutoSubmit(".release-calendar__filters .filters__release-type");
   }
+
+  function releaseTypeCensusAutoSubmit(formSelector) {
```

Comment:
Move function definition releaseTypeCensusAutoSubmit above line 35.
Move function call releaseTypeCensusAutoSubmit() on line 69 inside the test on line 35.
Remove lines 68-70